provider/plugin/plugin.go

```diff
@@ -28,9 +28,45 @@ import (
 	"sigs.k8s.io/external-dns/endpoint"
 	"sigs.k8s.io/external-dns/plan"
 
+	"github.com/prometheus/client_golang/prometheus"
 	log "github.com/sirupsen/logrus"
 )
 
+var (
+	recordsGauge = prometheus.NewGauge(
```

Comment:
Suggestion: Can you add last_successful_update / last_successful_change timestamp metrics as well?

Reply:
Yes, can do.

Reply:
@mrueg as a relatively noob with prometheus, I am not sure if this is the right recommendation. I was reading https://groups.google.com/g/prometheus-developers/c/Z5KwE6KbwN0 and I'd love to hear what you think about it or have more information from you before adding the additional metric.